conf.json

```diff
@@ -0,0 +1,10 @@
+{ map: 
```

Comment:
Why do you have map here, while on the following configuration file you use "settings"?: https://github.com/benchflow/spark-tasks-sender/blob/dev/spark-task-sender/config/config.json#L2